src/Utils.js

```diff
@@ -0,0 +1,48 @@
+import $ from 'jquery';
+import { proxy, minYear, maxYear, defaultYear } from './config';
+
+function parseHtml(html) {
+  let parser = new DOMParser();
+  return parser.parseFromString(html, 'text/html');
+}
+
+exports.fetchDocument = function(url) {
+  return fetch(url)
+    .then((response) => response.json())
+    .then(html => parseHtml(html))
+}
+
+exports.parseInput = function() {
+  let inputYear = $('#inputYear').val();
+  let errorMess = $('.errorMess');
+  let noQuerryMess = $('.noQueryMess');
+  $('#inputYear').val('');
+
+  if (isNaN(Number(inputYear)) || inputYear.length === 0) {
```

Comment:
Agree, with notable exception that Number accepts floating point numbers too...
How about [`parse-year`](https://github.com/bendrucker/parse-year)?
@kronicker Internally it uses _stricter_ `parseInt` that addresses your concern.